src/Concordium/Client/Runner.hs

```diff
@@ -1851,6 +1851,21 @@ data ModuleDeployTransactionCfg =
 moduleDeployTransactionPayload :: ModuleDeployTransactionCfg -> Types.Payload
 moduleDeployTransactionPayload ModuleDeployTransactionCfg {..} = Types.DeployModule mdtcModule
 
+-- |Checks if the contract has the given receive name or otherwise a fallback receive for
+-- v1 cobtracts. Returns the receive name when applicable. 
+checkAndGetContractReceiveName :: CI.ContractInfo -> Text -> IO Text
+checkAndGetContractReceiveName contrInfo receiveName = do
+  if CI.hasReceiveMethod receiveName contrInfo
+    then return (receiveName)
+    else
+      if not $ CI.hasFallbackReceiveSupport contrInfo
+        then logFatal [[i|The contract does not have an entrypoint named '#{receiveName}'.|]]
```

Comment:
I think this should be `logWarn` with a confirmation prompt.

We usually do this in concordium client to allow for testing, or if the user really wants to. So we prompt the user, telling them that what they want to do is almost certainly going to lead to failure, but allowing them to continue.

Reply:
done